lib/ui/common/components/cobble_button.dart

```diff
@@ -33,7 +33,7 @@ class CobbleButton extends StatelessWidget {
     Widget child = Row(
       mainAxisSize: MainAxisSize.min,
       children: [
-        if (icon is IconData) Icon(icon, size: 21),
+        if (icon is IconData) Icon(icon, size: 25),
```

Comment:
Default icon size is already 25, as defined here:
https://github.com/pebble-dev/mobile-app/blob/master/lib/ui/theme/cobble_theme.dart#L149
I tried removing `size: 25` in my local branch and indeed it takes global value. I think it's better for this PR to simply remove both parameters and allow global theme to defined icon size.

Reply:
Yeah, that does seem like a better option, I will fix that right away